Tests/OgreMain/src/VectorTests.cpp

```diff
@@ -108,4 +109,45 @@ TEST(VectorTests,PerpendicularVector)
     EXPECT_EQ((v3 + v1).dotProduct((v3 + v1).perpendicular()), (Real)0);
 }
 
+TEST(VectorTest, Matrix4ArrayLoading)
+{
+    Real arr0[16] = { 1, 0, 0, 0,
+                      0, 1, 0, 0,
+                      0, 0, 1, 0
```

Comment:
missing `,`

Reply:
and there's a missing "=" later too. Should be fixed.